packages/rest/src/rest.server.ts

```diff
@@ -242,8 +246,13 @@ export class RestServer extends Context implements Server, HttpServerLike {
    */
   protected _setupRouterForStaticAssets() {
     if (!this._routerForStaticAssets) {
-      this._routerForStaticAssets = express.Router();
-      this._expressApp.use(this._routerForStaticAssets);
+      this._routerForStaticAssets = express.Router() as ExpressRouter;
```

Comment:
I don't think this is a valid cast as `express.Router()` returns a function which is a handler.